setup.py

```diff
@@ -90,6 +90,7 @@
         "wrapt~=1.10",
         "aenum",
         'windows-curses;platform_system=="Windows"',
+        "filelock",
```

Comment:
Should be added to neovi extra requirement as well.

Reply:
Done

Reply:
Isn't that kind of duplicated if it's here and there?